app/styles/organisms/carousel.less

```diff
@@ -6,14 +6,24 @@
   gap: var(--spacing-px-12);
   position: relative;
 
+  &__reverse {
+    flex-direction: column-reverse;
+  }
+
   .page-btn {
     padding: var(--spacing-px-3);
     color: var(--color-gray-200);
-    font-size: 6px;
+    font-size: 8px;
+
+    i {
+      border: 1px solid var(--color-primary-100);
+      overflow: hidden;
+      border-radius: 999px;
+    }
```

Comment:
if we change this we will change how they already are displayed in the side-panel.
Let's not change the visuals of the indicators.

Reply:
The task mentions that there is a single usage in the soft (the sidepanel) & that we are safe to change the visuals of the `controls` not the indicators.

I understand that you checked the new visuals and are trying to match them though - but I'd rather have the product / design team update their own design based on the existing component.